src/hotspot/os_cpu/bsd_x86/os_bsd_x86.cpp

```diff
@@ -454,10 +455,10 @@ JVM_handle_bsd_signal(int sig,
 
   JavaThread* thread = NULL;
   VMThread* vmthread = NULL;
-  if (os::Bsd::signal_handlers_are_installed) {
+  if (PosixSignals::are_signal_handlers_installed()) {
     if (t != NULL ){
       if(t->is_Java_thread()) {
-        thread = t->as_Java_thread();
+        thread = (JavaThread*)t;
```

Comment:
Mis-merge? Please put this back to t->as_Java_thread()

Reply:
Merge issue, fixed.